docs/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+Solves issue #?
+
+# Description
+
+
+
+
+---
+text below is for instruction and can be removed
+
+# New node types
+Typically new node-types require an update of:
+
+- `core\src\create.jl`
+- `core\src\validation.jl`
+- `core\src\solve.jl`
+
+Updating other jl-files may depend on needs
+
+
+## ribasim_python
+- `newnodetype.py` with associated implementation in `python\ribasim\ribasim`.
+- add/update nodetype to `python\ribasim\ribasim\model.py`
+- add/update nodetype to `python\ribasim\tests\conftest.py`
+- add/update nodetype to `python\ribasim_api\tests\conftest.py`
+
+## documentation
+
+- update `docs\core\equations.qmd`
+- update `docs\core\usage.qmd`
+- update `docs\python\examples.ipynb`
```

Comment:
I would say there is a limit to which every new feature is crammed into the same example model. E.g. the current example model could be explicitly for all node types, and we start a new example model in a new notebook for control.